examples/large_models/segment_anything_fast/custom_handler.py

```diff
@@ -0,0 +1,91 @@
+import base64
+import io
+import json
+import logging
+import pickle
+import zlib
+
+import cv2
+import numpy as np
+import torch
+from PIL import Image
+from segment_anything_fast import SamAutomaticMaskGenerator, sam_model_registry
+from segment_anything_fast.tools import apply_eval_dtype_predictor
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class SegmentAnythingFastHandler(BaseHandler):
+    def __init__(self):
+        super(SegmentAnythingFastHandler, self).__init__()
+        self.mask_generator = None
+        self.initialized = False
+
+    def initialize(self, ctx):
+        properties = ctx.system_properties
+        if torch.cuda.is_available() and properties.get("gpu_id") is not None:
+            self.map_location = "cuda"
+            self.device = torch.device(
+                self.map_location + ":" + str(properties.get("gpu_id"))
+            )
+
+        model_type = ctx.model_yaml_config["handler"]["model_type"]
+        sam_checkpoint = ctx.model_yaml_config["handler"]["sam_checkpoint"]
+
+        self.model = sam_model_registry[model_type](checkpoint=sam_checkpoint)
+        self.model.to(self.device)
```

Comment:
self.device will not be set if cuda is unavailable

Reply:
Good catch. Added default as cpu. Not sure if we should just assert if no cuda?

Reply:
default cpu should be fine